node/config/v2_to_v3.go

```diff
@@ -0,0 +1,31 @@
+package config
+
+import (
+	"fmt"
+)
+
+// Migrate from config version 2 to version 3 (i.e. remove a few fields and add recently added fields)
+func v2Tov3(cfgPath string) (string, error) {
+	cfg, err := FromFile(cfgPath, DefaultBoost())
+	if err != nil {
+		return "", fmt.Errorf("parsing config file %s: %w", cfgPath, err)
+	}
+
+	boostCfg, ok := cfg.(*Boost)
+	if !ok {
+		return "", fmt.Errorf("unexpected config type %T: expected *config.Boost", cfg)
+	}
+
+	// For the migration from v1 to v2 just remove fields and update config version and add
+	// comments to the file
```

Comment:
I don't think we need this comment

Reply:
I think you may have removed the wrong comment 😆

Reply:
Fixed.